src/openvic-simulation/modifier/ModifierEffectCache.cpp

```diff
@@ -48,6 +49,12 @@ ModifierEffectCache::ship_type_effects_t::ship_type_effects_t()
 	evasion { nullptr },
 	torpedo_attack { nullptr } {}
 
+ModifierEffectCache::unit_terrain_effects_t::unit_terrain_effects_t()
+  : attack  { nullptr },
+  	defence  { nullptr },
```

Comment:
```suggestion
	defence  { nullptr },
```
Formatting (this line has two spaces before the tab, they should be removed)